### ISSUE TITLE
unlabeledX ?

### ISSUE THREAD

**Issue by bscully27** [tmadl/semisup-learn#12]
# label a few points 
labeled_N = 4
ys = np.array([-1]*len(ytrue)) # -1 denotes unlabeled point

Are unlabeled points a workaround for train/test data points?
Ie, model.fit(x_trn, y_trn).   model.score(x_tst, y_tst)
